runner.go

```diff
@@ -234,6 +242,10 @@ func filterExecuteTests(tests []*rainforest.RFTest, tags []string, forceExecute,
 	return result
 }
 
+func augmentJunitFileName(junitFile string, retryAttempt uint) string {
+	return fmt.Sprintf("%v.%v", junitFile, retryAttempt)
+}
```

Comment:
If we retry say three times, does this end up with files called:
* `foo.xml`
* `foo.xml.1`
* `foo.xml.1.2`

?

Reply:
Find/replace if `retryAttempt > 1` ?

Reply:
I'm guilty of regex.